crates/bitwarden-core/src/auth/auth_request.rs

```diff
@@ -36,11 +37,10 @@ pub(crate) fn new_auth_request(email: &str) -> Result<AuthRequestResponse, Crypt
     let spki = key.to_public_key().to_der()?;
 
     let fingerprint = fingerprint(email, &spki)?;
-    let b64 = STANDARD.encode(&spki);
 
     Ok(AuthRequestResponse {
-        private_key: STANDARD.encode(key.to_der()?),
-        public_key: b64,
+        private_key: key.to_der()?.as_ref().into(),
+        public_key: spki.as_ref().into(),
```

Comment:
Do we want a direct B64 -> Bytes and Bytes -> B64 conversion? It should avoid the need to call as_ref as much